README.md

```diff
@@ -11,64 +11,67 @@ zbox is a command line interface (CLI) tool to understand the capabilities of 0C
     - [Use custom miner/sharder](#use-custom-minersharder)
   - [Running zbox](#running-zbox)
-  - [Global Flags](#global-flags)
-  - [Commands](#commands)
-      - [Creating and Managing Allocations](#creating-and-managing-allocations)
-         - [Register wallet](#register-wallet)
-         - [Create new allocation](#create-new-allocation)
-         - [Free storage allocation](#free-storage-allocation)
-         - [Update allocation](#update-allocation)
-         - [Cancel allocation](#cancel-allocation)
-         - [Finalise allocation](#finalise-allocation)
-         - [Add curator](#add-curator)
-         - [Remove curator](#remove-curator)
-         - [Transfer allocation ownership](#transfer-allocation-ownership)
-         - [List blobbers](#list-blobbers)
-         - [Detailed blobber information](#detailed-blobber-information)
-         - [List all files](#list-all-files)
-         - [List owner's allocations](#list-owners-allocations)
-         - [Update blobber settings](#update-blobber-settings)
-       - [Uploading and Managing Files](#uploading-and-managing-files)
-         - [Upload](#upload)
-            - [Stream](#stream)
-            - [Feed](#feed)
-         - [Download](#download)
-         - [Update](#update)
-         - [Delete](#delete)
-         - [Share](#share)
-            - [share-encrypted revoke](#share-encrypted-revoke)
-         - [List](#list)
-         - [Copy](#copy)
-         - [Move](#move)
-         - [Sync](#sync)
-         - [Get differences](#get-differences)
-         - [Get wallet](#get-wallet)
-         - [Get](#get)
-         - [Get metadata](#get-metadata)
-         - [Download cost](#download-cost)
-         - [Upload cost](#upload-cost)
-         - [Rename](#rename)
-         - [Stats](#stats)
-         - [Repair](#repair)
-         - [Add collaborator](#add-collaborator)
-         - [Delete collaborator](#delete-collaborator)
-         - [Commit](#commit)
-         - [Sign data|](#sign-data)
-         - [Streaming](#streaming)
-      - [Lock and Unlock Tokens](#lock-and-unlock-tokens)
-        - [Challenge pool information](#challenge-pool-information)
-        - [Create read pool](#create-read-pool)
-        - [Read pool info](#read-pool-info)
-        - [Lock tokens into read pool](#lock-tokens-into-read-pool)
-        - [Unlock tokens from read pool](#unlock-tokens-from-read-pool)
-        - [Storage SC configurations](#storage-sc-configurations)
-        - [Stake pool info](#stake-pool-info)
-        - [Lock tokens into stake pool](#lock-tokens-into-stake-pool)
-        - [Unlock tokens from stake pool](#unlock-tokens-from-stake-pool)
-        - [Stake pools info of user](#stake-pools-info-of-user)
-        - [Write pool info](#write-pool-info)
-        - [Lock tokens into write pool](#lock-tokens-into-write-pool)
-        - [Unlock tokens from write pool](#unlock-tokens-from-write-pool)
+    - [Global Flags](#global-flags)
+- [Commands](#commands)
```

Comment:
Is this indent an intended changes or a mistake 

Reply:
it is automatically formatted by vscode.

Reply:
it seems be fine. just leave them as they are.

<img width="856" alt="Screen Shot 2022-03-23 at 20 55 17" src="https://user-images.githubusercontent.com/347651/159704112-8154a1a1-419a-46eb-9959-a20d803d738d.png">

Reply:
fixed. it is a new feature in vscode : auto-TOC based on markdown